### PR TITLE
BizOps onboarding

### PR DESCRIPTION
**Pull request by ebrodymoore**
Major WIP

### CODE REVIEW

**Comment by ebrodymoore** [comment]
Contents is basically ready for review but I'm still reviewing and making sure the reorg makes sense. Content is ready to be looked at though!

File: handbook/ops/bizops/analytics.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Analytics
+
+This page describes Sourcegraph's analytics function, our data sources, and how to use our data tools.
```
Comment (ebrodymoore) [inline handbook/ops/bizops/analytics.md:3]:
```suggestion
This page describes Sourcegraph's analytics function, our data sources, and how to use our data tools.
```

Write this instead

File: handbook/ops/bizops/index.md [hunk 2]
```diff
@@ -1,105 +1,40 @@
...
+**Internal BizOps projects:**  We use [GitHub issues in the Sourcegraph analytics repository](https://github.com/sourcegraph/analytics/issues) to document projects, and the [analytics kanban board](https://github.com/orgs/sourcegraph/projects/63) for team-wide prioritization and current projects.
 
-**Cross-functional projects:** Tag @ebrodymoore in existing [sourcegraph/sourcegraph GitHub issues](https://github.com/sourcegraph/sourcegraph/issues) or create a new issue. For larger projects, please work with the BizOps team to lay out the following information to help us understand and prioritize your request:
+**Cross-functional projects:** Tag @sourcegraph/bizops in existing [sourcegraph/sourcegraph GitHub issues](https://github.com/sourcegraph/sourcegraph/issues) or create a new issue in the BizOps repository. For larger projects, please work with the BizOps team to lay out the following information to help us understand and prioritize your request:
```
Comment (dadlerj) [inline handbook/ops/bizops/index.md:29]: Can you make "BizOps repository" a link here?

File: handbook/ops/bizops/index.md [hunk 3]
```diff
@@ -1,105 +1,40 @@
...
-
-### Quick links
-* [Sales/customer engineering](https://sourcegraph.looker.com/browse/boards/2)
-* [Product/engineering board](https://sourcegraph.looker.com/browse/boards/5)
-
-### Things to know about using Looker
-
-* By clicking `Explore from here` or changing a filter on a dashboard, you *will not* change the underlying dashboard. Unless you explicitly click `Edit`, you are considered to be on your own temporary branch and will not change anything (even for yourself the next time you open the dashboard). 
-* When creating and editing dashboards, save individual tables and charts as [looks](https://docs.looker.com/exploring-data/saving-and-editing-looks) instead of tiles directly to the dashboard. Looks can be added to multiple dashboards while tiles cannot be, and when look is edited, the changes will apply to dashboards that look exists. 
+To reach us, mention @business-team in the most relevant channel (i.e. #marketing for a marketing-related question).
```
Comment (dadlerj) [inline handbook/ops/bizops/index.md:38]:
```suggestion
To reach us, simply mention @business-team in the relevant channel or thread.
```

The wording here just felt a bit unclear.

File: handbook/ops/bizops/index.md [hunk 4]
```diff
@@ -1,105 +1,40 @@
...
 
-### Downsides of Looker (and our plans to address them)
-- **Discoverability of data**: Bookmarking, favoriting or adding the sales/customer engineering board, product/engineering board and server instances overview look (or some combination of them) to your Looker instance is the best solution right now. These are all kept up-to-date with the most relevant data for all teams. 
-- **Speed**: Looker’s UI makes it easy to analyze data, but the result really complex SQL query that take awhile to run (especially on dashboards that are compiled of many separate queries). Fixing the performance issues is not currently a priority, but is something that we’ll get to when we grow the team out.  
-- **Naming conventions**: We’re slowly working on making naming conventions of dashboards, graphs, data points, etc... more obvious. If you come across anything that isn’t clear, let us know! 
+If you need clarification or are not sure about any of the above, ask a question in #analytics or @business-ops channel.
```
Comment (dadlerj) [inline handbook/ops/bizops/index.md:40]:
```suggestion
If you need clarification or are not sure about any of the above, ask a question in #analytics or #business-ops channel.
```

File: handbook/ops/bizops/onboarding.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+You'll have to get some basics set up in your first few days:
+
+- Complete the [general onboarding](../../people-ops/onboarding/general_onboarding.md). 
+
+### Understand company and team mission, values and goals
+
+- [Sourcegraph FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb)
+- [Sourcegraph values](../../../company/values.md)
+- [Sourcegraph master plan](../../../company/strategy.md)
```
Comment (dadlerj) [inline handbook/ops/bizops/onboarding.md:17]:
```suggestion
- [Sourcegraph strategy](../../../company/strategy.md)
```

We actually don't use that term any more :)

File: handbook/ops/bizops/onboarding.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+### Get set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete the [general onboarding](../../people-ops/onboarding/general_onboarding.md). 
+
+### Understand company and team mission, values and goals
+
+- [Sourcegraph FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb)
```
Comment (dadlerj) [inline handbook/ops/bizops/onboarding.md:15]: Could you move this down to wherever seems like the best place in the list? I think it makes sense to start with the high-level pieces (values, strategy, etc.), and that the OKRs might be a bit confusing without that context)

File: handbook/ops/bizops/onboarding.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+- [Sourcegraph values](../../../company/values.md)
+- [Sourcegraph master plan](../../../company/strategy.md)
+- [Sourcegraph direction (1 year plan)](../../../direction/index.md)
+- [Current BizOps projects](https://github.com/orgs/sourcegraph/projects/63)
+- [BizOps @ Sourcegraph deck](https://docs.google.com/presentation/d/1B8TIkOsFXUofLKcwu3ZKUuFgSe7si_HNFXj8Qne4_ug/edit#slide=id.gb969c13690_0_15)
+
+### Understand Sourcegraph communication and documentation standards
```
Comment (dadlerj) [inline handbook/ops/bizops/onboarding.md:22]: This section is great. Did you write it from scratch, or pull it from another onboarding? We should share it if the former.

File: handbook/ops/bizops/onboarding.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+- We're a handbook-first company. Learn all about the handbook on its [handbook page](../../../handbook/usage.md). 
+- Join relevant Slack channels. The main channels are [listed in the handbook](../../communication/team_chat.md#channels), but it's a good idea to look through all the channels in our Slack and join all that seem relevant! The channels that BizOps uses most heavily are #business-ops and #analytics, in addition to the channels of teams we work with. 
+- Learn about how we work. We're in all-remote company, which means [we work asynchronously as much as possible](../../../company/remote/index.md#all-remote) and lot of decision making happens through [RFCs](../../communication/rfcs/index.md).
+
+### Onboarding sessions
```
Comment (dadlerj) [inline handbook/ops/bizops/onboarding.md:28]: Perhaps explain what this section means. Should they schedule these? How do they do so? Perhaps have links to the right people to connect with? Or just add a sentence at the top that says "your manager will help you schedule the following training sessions" or whatever.

File: handbook/ops/bizops/onboarding.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+    - Event logger and pings overview
+    - On-prem deployment to Google BigQuery to Looker data flow
+- Sales and marketing data overview
+- GitHub and git walk-through (Q&A)
+    - Read through the [git intro](../../people-ops/onboarding/git_intro.md) and [add yourself](../../people-ops/onboarding/general_onboarding.md#learn-how-to-contribute-to-the-handbook) to the team page
+
+### Get to know fellow teammates
+
+- Schedule 1:1s with teammates to start getting to know who you'll be working with! The [org chart](../../../company/team/org_chart.md) is a good place to start with this.
+
```
Comment (dadlerj) [inline handbook/ops/bizops/onboarding.md:47]:
```suggestion
- Begin joining relevant weekly team meetings wherever possible. For example, if you will be working closely with our commercial teams (e.g. Sales, Marketing, Customer Engineering), join our weekly Customer Ops sync, the company pipeline review, and more. Your manager will help you identify and get invites to the right list of meetings.

```

**Review by dadlerj — APPROVED** [review]
Thanks! You should wait for other reviews, but left a few comments here.

File: handbook/ops/bizops/analytics.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+We collect data from the following:
+
+- Google Analytics: Website analytics for Sourcegraph marketing and docs pages (not Sourcegraph.com)
+- HubSpot: Marketing automation
+- Salesforce: Customer Relationship Management system (CRM)
+- MixMax: Email marketing automation (Apollo is not used in production, but still retains data)
+- ZoomInfo:  Data enrichment of account and contact information
+- Sourcegraph.com Site-admin pages: customer subscriptions and license keys
+- [Pings](https://docs.sourcegraph.com/admin/pings) from self-hosted Sourcegraph instances containing anonymous and aggregated information. There are [specific guidelines](https://docs.sourcegraph.com/dev/background-information/adding_ping_data) that must be followed for teams to add ping data. 
+- [Custom tool to track events](https://github.com/sourcegraph/sourcegraph/issues/5486) on the Sourcegraph.com instance
```
Comment (attfarhan) [inline handbook/ops/bizops/analytics.md:20]: @ebrodymoore is this referring to eventLogger? 

If so, this collects events on _both_ local instances and sourcegraph.com (see the [backend function for logEvent](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/internal/usagestats/event_handlers.go?utm_source=VSCode-1.2.3#L42-54), which collects events for both). The difference is that Sourcegraph.com events go to BigQuery, whereas on an instance it goes to the local postgres database.

```suggestion
- [Event logger: custom tool to track events](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/web/src/tracking/eventLogger.ts). On Sourcegraph.com, this sends events directly to BigQuery. On customer instances, this sends events to the `EventLogs` database, which is then used to populate pings.
```

File: handbook/ops/bizops/onboarding.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+- Complete the [general onboarding](../../people-ops/onboarding/general_onboarding.md). 
+
+### Understand company and team mission, values and goals
+
+- [Sourcegraph FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb)
+- [Sourcegraph values](../../../company/values.md)
+- [Sourcegraph master plan](../../../company/strategy.md)
+- [Sourcegraph direction (1 year plan)](../../../direction/index.md)
+- [Current BizOps projects](https://github.com/orgs/sourcegraph/projects/63)
```
Comment (attfarhan) [inline handbook/ops/bizops/onboarding.md:19]:
```suggestion
- [Current BizOps projects](https://github.com/orgs/sourcegraph/projects/63)
- [CustomerOps project tracker](https://docs.google.com/spreadsheets/d/16YKiI5-jn5xH8t1n7z3pC-4AQfJnOsZLlgJfWj2RlY4/edit#gid=0)
```
@shaan-sourcegraph adding the link to the customer ops project tracker here. Can you make it accessible for all users with a @sourcegraph.com email? Can't currently access it.

File: handbook/ops/bizops/onboarding.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+- GitHub and git walk-through (Q&A)
+    - Read through the [git intro](../../people-ops/onboarding/git_intro.md) and [add yourself](../../people-ops/onboarding/general_onboarding.md#learn-how-to-contribute-to-the-handbook) to the team page
+
+### Get to know fellow teammates
+
+- Schedule 1:1s with teammates to start getting to know who you'll be working with! The [org chart](../../../company/team/org_chart.md) is a good place to start with this.
+
+## Weeks 2 and 3
+
+- Your manager and you will work together to find good onboarding projects that combine your areas of interest with real problems we need to solve. These projects should the following elements:
```
Comment (attfarhan) [inline handbook/ops/bizops/onboarding.md:50]:
```suggestion
- Your manager and you will work together to find good onboarding projects that combine your areas of interest with real problems we need to solve. These projects should include the following elements:
```

**Review by attfarhan — APPROVED** [review]

File: handbook/ops/bizops/analytics.md (hunk 10 above)
Comment (ebrodymoore) [reply handbook/ops/bizops/analytics.md:20]: Thanks! This actual content hasn't been updated in a looong time so please update anything else you see

File: handbook/ops/bizops/onboarding.md (hunk 7 above)
Comment (ebrodymoore) [reply handbook/ops/bizops/onboarding.md:22]: From scratch but probably with some inspiration from other team-specific onboardings. Honestly can't remember :)

**Pull request merged**